<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDiscountEdit</class>
 <widget class="QDialog" name="dlgDiscountEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="gbIdentification">
         <property name="title">
          <string> Identification </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledName">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="gbDiscountCategory">
           <property name="title">
            <string> Category </string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="rbRegularCustomer">
              <property name="text">
               <string>Regular customer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbEmployee">
              <property name="text">
               <string>Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbService">
              <property name="text">
               <string>Service person</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbGuest">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Guest</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbCompany">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Company</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbProduct">
              <property name="text">
               <string>Product</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbPaymentMethod">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Payment method</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbCoupon">
              <property name="text">
               <string>Coupon</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="gbDiscountCombo">
             <property name="title">
              <string>Guest / Company / Product ... </string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QComboBox" name="cmbHCDList">
                <property name="enabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QGroupBox" name="gbDiscountType">
               <property name="title">
                <string> Type </string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QRadioButton" name="rbDiscountValue">
                  <property name="text">
                   <string>fix value</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbDiscountPercent">
                  <property name="text">
                   <string>percent</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="gbValue">
               <property name="title">
                <string> Value </string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLineEdit" name="ledDiscount">
                  <property name="minimumSize">
                   <size>
                    <width>70</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="pbSave">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkIndependent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Independent from the local studio.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSave</sender>
   <signal>clicked()</signal>
   <receiver>dlgDiscountEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgDiscountEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
